### PR TITLE
Add audio and video answers to the API docs

### PR DESCRIPTION
**Pull request by robvitaro**
Refer to [https://github.com/indeedassessments/ia-web/pull/464](https://github.com/indeedassessments/ia-web/pull/464)

**Probably should not be merged until the above PR is settled. There may be properties to add/remove still**

### CODE REVIEW

File: sections/attempts.md [hunk 1]
```diff
@@ -209,6 +209,76 @@ Endpoints:
...
+              "retakes": 0,
+              "time_limit_in_seconds": 60
+            }
+          }
+        ],
+        "answers": [
+
+        ],
+        "step": {
+        "name": "A video interview step",
```
Comment (juandazapata) [inline sections/attempts.md]: Indentation looks wrong here.

File: sections/assignments.md [hunk 2]
```diff
@@ -490,6 +490,76 @@ curl -s -H "Authorization: token $API_KEY" \
...
+                  "retakes": 0,
+                  "time_limit_in_seconds": 60
+                }
+              }
+            ],
+            "answers": [
+
+            ],
+            "step": {
+            "name": "A video interview step",
```
Comment (juandazapata) [inline sections/assignments.md]: Indentation looks wrong here.

**Review by juandazapata — APPROVED** [review]
I'm gonna approve this, but you should fix the indentation issues before merging, since it makes the code more readable.

**Review by sonnym — APPROVED** [review]
💯

**Pull request merged**